src/cobald/decorator/logger.py

```diff
@@ -127,7 +127,8 @@ def __init__(
             raise RuntimeError(
                 f"invalid {type(self).__name__} message field: {e}"
             ) from None
-        self._logger = None  # type: logging.Logger
-        self.message = message
+        # set properly by self.name setter immediately afterwards
+        self._logger: logging.Logger = None  # type: ignore
```

Comment:
Should it not be 

```suggestion
        self._logger: Optional[logging.Logger] = None  # type: ignore
```
instead?

Reply:
Thanks for questioning this. The answer is yes and no. Technically it is `None` for a moment, but never actually visible like this.

I've changed it now to only be the type hint without an initialisation. I was paranoid that this wasn't acceptable for Py3.6 and inspection, but apparently it works fine.